<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="AppName">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>361</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>App Name</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="LabelsList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>256</width>
      <height>411</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Labels:</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="AddLabels">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Add Labels, Separate using comma</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Entertainment">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>310</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Entertainment</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Productivity">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>370</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Productivity</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Others">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>430</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Others</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Save">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>610</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Back">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>610</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Rename">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Rename This App?</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Block">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>490</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Block</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
